docs/source/custom_datasets.rst

```diff
@@ -0,0 +1,605 @@
+Fine-tuning with custom datasets
+================================
+
+This tutorial will take you through several examples of using 🤗 Transformers models with your own datasets. The
+guide shows one of many valid workflows for using these models and is meant to be illustrative rather than
+definitive. We show examples of reading in several data formats, preprocessing the data for several types of tasks,
+and then preparing the data into PyTorch/TensorFlow ``dataset`` objects which can easily be used either with
+:class:`~transformers.Trainer`/:class:`~transformers.TFTrainer` or with native PyTorch/TensorFlow.
+
+We include several examples, each of which demonstrates a different type of common downstream task:
+
+  - :ref:`seq_imdb`
+  - :ref:`tok_ner`
+  - :ref:`qa_squad`
+  - :ref:`resources`
+
+Note: Many of the datasets used in this tutorial are available and can be more easily accessed using
+`🤗 NLP library <https://github.com/huggingface/nlp>`_. 🤗 Transformers also includes a number of data
+:doc:`processors <main_classes/processors>` for working with specific datasets like squad and glue. Neither is used
```

Comment:
I wouldn't reference the processors as I think the end goal is to remove them, once we use nlp in every example.